scripts/setStages.ts

```diff
@@ -27,6 +30,8 @@ export const setStages = async (
   ) as StageConfig[];
   const ERC721M = await ethers.getContractFactory(ContractDetails.ERC721M.name);
   const contract = ERC721M.attach(args.contract);
+  const gasLimit = args.gaslimit ? args.gaslimit : 500_000;
+  const gasPrice = args.gasprice ? args.gasprice * 1e9 : 500 * 1e9;
```

Comment:
Why multiplying by 1e9?

Reply:
The input param is set in Gwei, I'm converting Gwei to Wei.

Reply:
I think both are in Wei per your argument description above?